dut/doc.go

```diff
@@ -0,0 +1,66 @@
+// dut manages Devices Under Test (a.k.a. DUT) from a host.
```

Comment:
this looks like a copy of the cmds/dut/doc.go.  from glancing at the Changed Files, it looks like dut/ and cmds/dut/ are the same.  not sure which one you want to keep.